Fanny/Fanny.xcodeproj/project.pbxproj

```diff
@@ -658,15 +658,15 @@
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = E5BK5K7RRN;
+				DEVELOPMENT_TEAM = HT8VTXFQ39;
 				ENABLE_HARDENED_RUNTIME = YES;
 				INFOPLIST_FILE = fanny/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/../Frameworks",
 				);
 				MACOSX_DEPLOYMENT_TARGET = 10.13;
-				MARKETING_VERSION = 2.1.0;
+				MARKETING_VERSION = 2.1.1;
```

Comment:
We'd want to version the project separately from a PR.

Reply:
Makes sense. I was not sure about the conventions here, if you bounce it upon completing a new version or when you start adding things